protobuf/proto-types/validation.proto

```diff
@@ -15,6 +15,6 @@ message BlockConnected {
   required int64    transactions            = 3;  // Number of transactions in the connected block.
   required int32    inputs                  = 4;  // Number of inputs in the connected block.
   required int64    sigops                  = 5;  // Number of sigops in the connected block.
-  required int64    connection_time         = 6;  // Time it took to connect the block in microseconds (µs).
+  required uint64   connection_time         = 6;  // Time it took to connect the block in nanoseconds (ns).
```

Comment:
drop the `as i64` from https://github.com/0xB10C/peer-observer/blob/8a49c622e21a14a08f4d40df8f16bfaf5830e874/shared/src/validation.rs#L16 too